hash/gbcolor.xml

```diff
@@ -6244,14 +6474,15 @@ Unreleased (music source code exists, possibly no prototypes exist)
 	</software>
 
 	<software name="onepyume">
-		<!-- Notes: SGB enhanced -->
 		<description>From TV Animation One Piece - Yume no Luffy Kaizokudan Tanjou! (Jpn, Rev. A)</description>
 		<year>2001</year>
 		<publisher>Banpresto</publisher>
 		<info name="serial" value="DMG-BLUJ-JPN"/>
+		<info name="gold" value="20010615"/>
 		<info name="release" value="20010427"/>
-		<info name="alt_title" value="フロム・テレビ・アニメーション・ワンピース 夢のルフィ海賊団誕生!"/>
+		<info name="alt_title" value="FROM TV ANIMATION　ＯＮＥ ＰＩＥＣＥ　夢のルフィ海賊団誕生！"/>
```

Comment:
Do you have a source for “ONE PIECE” being fullwidth?  All the images of it use the regular logo which uses a proportional font.

Reply:
I just copied and pasted it from the official spreadsheets.

Reply:
I've changed it to regular width.